### PR TITLE
Update dependency on boost.

### PR DESCRIPTION
**Pull request by ProgVal**
I did this years ago and don't remember why. It may have
to do with making the server work with Debian 8 or 9.

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -285,7 +295,7 @@ def main():
     env.Append(CPPPATH=['#libusl/src', '#'])
     env.Append(CXXFLAGS=' -Wall')
     env.Append(LINKFLAGS=' -Wall')
-    env.Append(LIBS=['SDL_net'])
+    env.Append(LIBS=['SDL_net', 'pthread'])
```
Comment (stephanemagnenat) [inline SConstruct:298]: I'm worried that this would break compilation on Windows.

**Comment by stephanemagnenat** [comment]
I will wait for the SDL2 port to land, and then update Scons to Python3, and then see whether this change is still relevant.

**Comment by stephanemagnenat** [comment]
@ProgVal I merged the update to Python3 and now Glob2 compiles on Ubuntu 20.04. Do you think this change is still necessary or desired?

**Comment by ProgVal** [comment]
Just tried it and it builds and seems to work, thanks :)

The current git master is now running on the YOG server.

**Pull request closed without merging**